lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -22,12 +22,15 @@
 using Microsoft.Azure.Batch.Common;
 using Org.Apache.REEF.Client.AzureBatch.Parameters;
 using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Utilities.Logging;
 using BatchSharedKeyCredential = Microsoft.Azure.Batch.Auth.BatchSharedKeyCredentials;
 
 namespace Org.Apache.REEF.Client.DotNet.AzureBatch
 {
     public class AzureBatchService : IDisposable
     {
+        private static readonly Logger Logger = Logger.GetLogger(typeof(AzureBatchService));
```

Comment:
Why remove the logging?

Reply:
will change variable name "Logger" to "LOGGER". Not sure what does it mean here "remove logging".